index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>personal portfolio</title>
+</head>
+
+<body>
+    <div class="container">
+        <header class="header-flex align-header">
```

Comment:
- [ ] It would be better if we apply the margin `top`, `bottom`, `left`, `right` to achieve the required design as in the screenshot below. Also, let us note that we are required to achieve the same design
![Screenshot from 2021-09-16 09-26-11](https://user-images.githubusercontent.com/46388113/133587737-9ada25bb-5b10-4b6a-9c0c-e60cf697e391.png)
### Here is ours
![Screenshot from 2021-09-16 09-29-45](https://user-images.githubusercontent.com/46388113/133587882-60b76c04-1cbf-4300-8b06-4f57cbab16d0.png)
As we can see we have padding and margins to apply to achieve this design and some contents are missing